backend/endpoints/messages.py

```diff
@@ -209,7 +210,7 @@ async def update_message(
 
 @router.get(
     "/org/{org_id}/rooms/{room_id}/messages",
-    response_model=ResponseModel,
+    response_model=Page[Message],
```

Comment:
boss @man-of-mind abeg how is this used?

Reply:
It's to give custom response model based on the pagination, it's pydantic model that represent the result or response.